packages/cosmos_ui_components/lib/components/cosmos_text_field.dart

```diff
@@ -69,9 +72,9 @@ class _CosmosTextFieldState extends State<CosmosTextField> {
       decoration: InputDecoration(
         border: UnderlineInputBorder(borderSide: BorderSide(color: theme.colors.inputBorder)),
         hintText: widget.hint,
-        hintStyle: CosmosTextTheme.copy0Normal,
-        counterText: '',
```

Comment:
Removing this will cause the character length to appear below the text field in cases where `maxLength` is provided